launch_cluster/launch.py

```diff
@@ -342,7 +330,7 @@ def validate_config():
     if config.ignore_permissions == "false":
         permissions = oct(os.stat(KEY_PATH).st_mode % 2 ** 9)
         #if permissions[2:] != "600":   <--- And update this
-        if permissions[1:] != "600":
+	if permissions[1:] != "600":
```

Comment:
Its hard to tell by the naked eye here, but is this indented correctly?

Reply:
Good catch. No, it is not. Accordingly, I have fixed it in 787fd636c74c12d3f2b192a6b06d2120a0d0b900.

Reply:
Sure thing. I have fixed it. Thanks.